internal/godoc/encode_ast.gen.go

```diff
@@ -212,6 +212,89 @@ func encode_ast_BadExpr(e *codec.Encoder, x *ast.BadExpr) {
 	e.EndStruct()
 }
 
+func init() {
+	codec.Register(&ast.OverloadFuncDecl{},
```

Comment:
No need to change this, since a gop project should be also a go project. we just need to adjust packages after parsing based on gop rules.

I am thinking the best position to do this is https://github.com/goplus/pkgsite/blob/df601fe449072963fe202be93c21a664c510f633/internal/fetch/load.go#L305